.github/main.yml

```diff
@@ -19,10 +19,16 @@ jobs:
         uses: subosito/flutter-action@v2
         with:
           flutter-version: 3.19.5
```

Comment:
Update to `flutter-version: 3`

Reply:
Could we focus on if the code is building or running locally first?